tools/api-markdown-documenter/api-report/api-markdown-documenter.public.api.md

```diff
@@ -275,13 +275,13 @@ export function documentationNodeToHtml(node: DocumentationNode, config: ToHtmlC
 export function documentationNodeToHtml(node: DocumentationNode, context: ToHtmlContext): Nodes;
 
 // @public
-export interface DocumentationParentNode<TDocumentationNode extends DocumentationNode = DocumentationNode> extends Parent<TDocumentationNode, Data>, DocumentationNode {
+export interface DocumentationParentNode<TDocumentationNode extends Node_2> extends Parent<TDocumentationNode, Data>, DocumentationNode {
```

Comment:
Should we do something in the type definitions to prevent the `Node_2`? These tend to indicate implicit types where we could do explicit, IIRC?

Reply:
In this case, I _suspect_ this aliasing is being done because `mdast`, `hast`, and `unist` all have their own types called `Node` that refer to one another. I can dig into this a bit more in subsequent PRs though. I'm pretty comfortable leaving this be for now, because this isn't new aliasing.